docs/website/content/docs/quickstart.md

```diff
@@ -7,6 +7,9 @@ draft: false
 bref: "Create your first diagram from a Sysl specification"
 toc: false
 ---
+
+Sysl consists of two executables: `sysl` and `reljam`.</br> The **Sy**stem **S**pecification **L**anguage `sysl` is mainly concerned with diagram creation whereas the **Rel**ational **Ja**va **M**odel program `reljam` generates different types of source code output.
+
 Install Sysl
 ------------
 Windows users can run standalone Sysl executables whereas users of other operating systems need to work with either Python or Docker.
```

Comment:
Section on Docker needs to be enhanced.
1. Add -v command to mount the volume
2. Add section on to pull plantuml docker image, run the image on 8080. also change SYSL_PLANTUML to this local image.

Reply:
hmm, you have got issue #121  (Document docker usage) assigned to you. once you have done it for the Readme.rst I'm happy to transfer it to the website (which is really what I've done here too).